src/test/java/nextstep/favorite/acceptance/FavoriteSteps.java

```diff
@@ -0,0 +1,42 @@
+package nextstep.favorite.acceptance;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.MediaType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FavoriteSteps {
+    public static ExtractableResponse<Response> 즐겨찾기_생성_요청(Long source, Long target, String accessToken) {
+        Map<String, Object> request = new HashMap<>();
+        request.put("source", source);
+        request.put("target", target);
+
+        return RestAssured.given().log().all()
+                .auth().oauth2(accessToken)
+                .body(request)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().post("/favorites")
+                .then().log().all()
+                .extract();
+    }
+
+    public static ExtractableResponse<Response> 즐겨찾기_조회_요청(String accessToken) {
+        return RestAssured.given().log().all()
+                .auth().oauth2(accessToken)
+                .when().get("/favorites")
+                .then().log().all()
+                .extract();
+    }
+
+    public static ExtractableResponse<Response> 즐겨찾기_삭제_요청(String accessToken, Long favoriteId) {
+        return RestAssured.given().log().all()
+                .auth().oauth2(accessToken)
+                .when().delete("/favorites/" + favoriteId)
```

Comment:
delete 메서드 등은 pathParams를 지원하니 활용해 보셔도 좋을 것 같아요
```suggestion
                .when().delete("/favorites/{favoriteId}", favoriteId)
```